src/app/features/videomania/index.js

```
@@ -124,6 +215,26 @@ const videoManiaSlice = createSlice({
+      })
+      .addCase(likeUnlikeVideo.fulfilled, (state, action) => {
+        state.isLoading = false;
+      })
+      .addCase(likeUnlikeVideo.rejected, (state, action) => {
+        state.isLoading = false;
+        state.error = action?.payload;
+      })
+      .addCase(addCommentOnVideo.pending, (state) => {
+        state.isLoading = true;
+      })
+      .addCase(addCommentOnVideo.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)